pkg/cnab/provider/install.go

```diff
@@ -35,7 +35,7 @@ func (d *Runtime) Install(args ActionArguments) error {
 	}
 	d.Extensions = exts
 
-	params, err := d.loadParameters(c, args.Params, args.ParameterSets, string(manifest.ActionInstall))
+	params, err := d.loadParameters(c, args.Params, string(manifest.ActionInstall))
```

Comment:
Oh nice, this is much cleaner to have already resolved all parameters by the time we get to the runtime here.